docs/core-api/BITSWAP.md

```diff
@@ -16,22 +16,21 @@
   - [Returns](#returns-2)
   - [Example](#example-2)
 
-## `ipfs.bitswap.wantlist([peerId,] [options])`
+## `ipfs.bitswap.wantlist([options])`
 
-> Returns the wantlist, optionally filtered by peer ID
+> Returns the wantlist for your node or for a connected peer
 
 ### Parameters
 
-| Name | Type | Description |
-| ---- | ---- | ----------- |
-| peerId | [PeerId][], [CID][], `String` or `Buffer` | An optional peer ID to return the wantlist for |
+None
 
 ### Options
 
 An optional object which may have the following keys:
 
 | Name | Type | Default | Description |
 | ---- | ---- | ------- | ----------- |
+| peer | [PeerId][], [CID][], `String` or `Buffer` | A peer ID to return the wantlist for |
```

Comment:
This is an array of peers ? and the default can probably be more descriptive like Self Peer ID or something.

Reply:
No, it's a Markdown link that's in the footer of the document.

Reply:
Like this: https://www.markdownguide.org/basic-syntax/#reference-style-links

Reply:
got it, but the default is the node's peerid right ?

Reply:
In effect, but there are actually two different methods we call in the bitswap module: https://github.com/ipfs/js-ipfs/blob/master/packages/ipfs/src/core/components/bitswap/wantlist.js#L15-L16

We could split this into two methods at the Core API level too?

Reply:
I don't we need to do that, not much to be gained there.
Just need to make it clear that if the user doesn't provide a peer in the options the node's peer id is used

Reply:
You know, the more I think about it, the more I think it's a separate operation.

Reply:
I've made it a separate operation.